components/KopierenButton.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export default function StyledKopierenButton(props) {
```

Comment:
I would not mix english and german...
```suggestion
export default function StyledCopyButton(props) {
```